src/erlfmt.erl

```diff
@@ -22,15 +22,18 @@
     read_forms/1,
     read_forms_string/2,
     format_error/1,
-    format_error_info/1
+    format_error_info/1,
+    contains_pragma_forms/1
 ]).
 
--export_type([error_info/0, out/0]).
+-export_type([error_info/0, out/0, config/0]).
 
 -type error_info() :: {file:name_all(), erl_anno:location(), module(), Reason :: any()}.
 
 -type out() :: standard_out | {path, file:name_all()} | replace.
 
+-type config() :: {RequirePragma :: boolean(), Out :: out()}.
```

Comment:
This config type is created to make space for InsertPragma in a following pull request

Reply:
Makes sense, took me a while to see:

no require, no insert => current
no require, insert => format all files, but insert pragma after formatting.
require, no insert => only format files with pragma
insert, require => Ah, I see this option doesn't make sense